modules/lang-painless/src/yamlRestTest/resources/rest-api-spec/test/painless/50_script_doc_values.yml

```diff
@@ -1696,3 +1784,202 @@ setup:
   - match: { hits.hits.1.fields.f_list.0: "dne" }
   - match: { hits.hits.1.fields.f_list2.0: "789" }
   - match: { hits.hits.1.fields.all.0: "10111213789876deflmnrstwyz" }
+
+---
+"unsupported date methods":
+
+  - do:
+      catch: bad_request
+      search:
+        rest_total_hits_as_int: true
+        body:
+          query: { term: { _id: 1 } }
+          script_fields:
+            field:
+              script:
+                source: "/* avoid stash */ $('date', null).getCenturyOfEra()"
+  - match: { error.failed_shards.0.reason.caused_by.type: "unsupported_operation_exception" }
+  - match: { error.failed_shards.0.reason.caused_by.reason:
+               "joda method [getCenturyOfEra] is no longer available; use [<ZonedDateTime instance>.get(ChronoField.YEAR_OF_ERA) / 100] instead" }
```

Comment:
For each of these suggestions, can you add a test that _uses_ the suggestion right the test that suggests the alternative?

Reply:
Good call. Caught a pair of bugs for the week based suggestions!